lib/board.rb

```diff
@@ -0,0 +1,55 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def reset!
+    self.cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def display
+    puts "------------"
+    puts " #{@cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "------------"
+    puts " #{@cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "------------"
+    puts " #{@cells[6]} | #{cells[7]} | #{cells[8]} "
+    puts "------------"
+  end
+
+  def position(cell)
+    self.cells[cell.to_i - 1]
+  end
+
+  def cell_taken?(cell)
+    cell == "X" || cell == "O"
+  end
```

Comment:
Couldn't you implement this logic directly in `taken?`? Why abstract it out to an additional method?